runtime/jcl/common/jdk_internal_misc_ScopedMemoryAccess.cpp

```diff
@@ -86,6 +86,9 @@ Java_jdk_internal_misc_ScopedMemoryAccess_closeScope0(JNIEnv *env, jobject insta
 					setHaltFlag(walkThread, J9_PUBLIC_FLAGS_CLOSE_SCOPE);
 					walkThread->scopedError = errorObj;
 					walkThread->closeScopeObj = closeScopeObj;
+
+					Assert_JCL_true(vm->closeScopeNotifyCount >= 0);
```

Comment:
I dont think this assert is needed, unsinged int guarantees that for us

Reply:
I dont think I follow, isnt `(U_64)-1 == U_64_MAX`?

Reply:
I misinterpreted. You're correct, we can remove this assert.

Reply:
Assertion has been removed.